content/en/docs/contribute/localization.md

```diff
@@ -23,9 +23,9 @@ the docs for a different language.
 
 You can help add or improve the content of an existing localization. In
 [Kubernetes Slack](https://slack.k8s.io/), you can find a channel for each
-localization. There is also a general [SIG Docs Localizations Slack
-channel](https://kubernetes.slack.com/messages/sig-docs-localizations) where you
-can say hello.
+localization. There is also a general
+[SIG Docs Localizations Slack channel](https://kubernetes.slack.com/messages/sig-docs-localizations)
+where you can say hello.
```

Comment:
is this editorconfig thing, I dont see how this improves the content

Reply:
Our script /scripts/linkchecker.py cannot handle links that span multiple lines.